admin-manual/installation/installation.rst

```diff
@@ -40,12 +40,17 @@ Technical Requirements
 
 **Operating System**
 
-Archivematica 1.6.0 has been tested with:
+Archivematica 1.6.1 installation instructions are provided here for the 
```

Comment:
I am not sure that it is a good idea to use a variable here.  For example, there is a sentence in this rst file that says:

"Archivematica 1.6.1 requires ElasticSearch 1.x" 

It may not necessarily be valid to change that to say 1.7.0.  The text still has to be reviewed and updated with the new release, to make sure it is accurate.

In some places, that